Userland/Libraries/LibGL/GLContext.cpp

```diff
@@ -629,6 +631,17 @@ void GLContext::gl_enable(GLenum capability)
     bool update_rasterizer_options = false;
 
     switch (capability) {
+    case GL_CLIP_PLANE0:
+    case GL_CLIP_PLANE1:
+    case GL_CLIP_PLANE2:
+    case GL_CLIP_PLANE3:
+    case GL_CLIP_PLANE4:
+    case GL_CLIP_PLANE5: {
+        auto pl_idx = static_cast<size_t>(capability) - GL_CLIP_PLANE0;
```

Comment:
```suggestion
        auto plane_idx = static_cast<size_t>(capability) - GL_CLIP_PLANE0;
```

Don't be afraid to use a long variable name, we don't restrict line length as long as it's not unreasonably long :^)